### ISSUE TITLE
Vite plugin doesn't seem to work with official Vite scaffold

### ISSUE THREAD

**Issue by dearlordylord** [ryoppippi/unplugin-typia#157]
After installing per Vite docs https://vitejs.dev/guide/#scaffolding-your-first-vite-project , I'm adding UnpluginTypia to vite.config.ts.

It seems to get loaded (its message about cache pops up on `npm run dev`), but it reports that "transform is not configured" in the console and crashes the app.

I successfully ran your example with react+bun though, including with copy-paste of my type/runtime code.

Reproduction and full description (please refer for error message, steps of reproduction; you also can just run this repo to reproduce it): https://github.com/dearlordylord/typia-vite

**Comment by ryoppippi** [comment]
OK, so when I delete your `package-lock.json` and install npm again, it works fine

**Comment by dearlordylord** [comment]
@ryoppippi thank you for looking into it

I went with merging/checking out your changes, then

- `rm -rf node_modules/.cache/unplugin_typia`
- `rm package-lock.json`
- `npm i` which updated package-lock.json to a different content than in the repo

it ended up with the same error, with no extra debug info

(`npm --version` is 10.2.4)

then, I nuked node_modules and did `npm i` again

the error / debug output stays the same

the branch https://github.com/dearlordylord/typia-vite/blob/npm-i-after-fix-1/package-lock.json has the updated package-lock

I've also installed `vite-plugin-inspect` to check it but I'm not sure what I should be looking for (checked in into the same branch)

finally, resetting cache `rm -rf node_modules/.cache/unplugin_typia` again doesn't seem to have any effect

**Comment by ryoppippi** [comment]
So, I uploaded this to vercel
[typia-vite.vercel.app](https://typia-vite.vercel.app/)
https://github.com/ryoppippi/typia-vite
As you can see, it works fine.

**Comment by dearlordylord** [comment]
<img width="1624" alt="Screenshot 2024-07-01 at 1 46 07 PM" src="https://github.com/ryoppippi/unplugin-typia/assets/947595/d02c6b71-58dd-4463-b9bf-52f13b4854a6">

from the PR checked out and link http://localhost:5173/__inspect/#/module?id=/private/tmp/typia-vite/src/App.tsx&index=1

that's a bummer that it seems it doesn't work only for me! 

I'll try to deploy the code on my private server / will try to tackle it further, I'll let you know if I find anything useful

**Comment by dearlordylord** [comment]
I ran it on my Dokku with no issues. I noticed it uses a bit older Node version so downgraded to it locally, cleaned cache and ran again and it worked!
I tried to reproduce the issue upgrading node to what it was again (21.6.1), cleaned cache and it worked again...
So I have no clue what was it. Probably skill issues. 
The only difference that my node is installed with `n` now vs. it being installed with `brew` before
